tasks/postcss.ts

```diff
@@ -0,0 +1,76 @@
+export = function(grunt: IGrunt) {
+	const path = require('path');
+	const fs = require('fs');
+	const postCssImport = require('postcss-import');
+	const postCssNext = require('postcss-cssnext');
+	const postCssModules = require('postcss-modules');
+
+	grunt.loadNpmTasks('grunt-postcss');
+
+	const distDirectory = grunt.config.get<string>('distDirectory');
+
+	const moduleProcessors: any = [
+		postCssImport,
+		postCssNext({
+			features: {
+				autoprefixer: {
+					browsers: [
+						'last 2 versions',
+						'ie >= 10'
+					]
+				}
+			}
+		}),
+		postCssModules({
+			generateScopedName: '[name]__[local]__[hash:base64:5]',
```

Comment:
Was thinking we should just change this to a hash without the other information to minimise the `.css.json` and `.css` files. Any thoughts??